src/Mvc/Mvc.ViewFeatures.Prerendering/src/HtmlHelperComponentPrerenderingExtensions.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Server.Prerendering;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures
+{
+    /// <summary>
+    /// Extensions for rendering components.
+    /// </summary>
+    public static class HtmlHelperComponentPrerenderingExtensions
+    {
+        /// <summary>
+        /// Renders the <typeparamref name="TComponent"/> <see cref="IComponent"/>.
+        /// </summary>
+        /// <param name="htmlHelper">The <see cref="IHtmlHelper"/>.</param>
+        /// <returns>The HTML produced by the rendered <typeparamref name="TComponent"/>.</returns>
+        public static Task<IHtmlContent> PrerenderComponentAsync<TComponent>(this IHtmlHelper htmlHelper) where TComponent : IComponent
+        {
+            if (htmlHelper == null)
+            {
+                throw new ArgumentNullException(nameof(htmlHelper));
+            }
+
+            return htmlHelper.PrerenderComponentAsync<TComponent>(null);
+        }
+
+        /// <summary>
+        /// Renders the <typeparamref name="TComponent"/> <see cref="IComponent"/>.
```

Comment:
XML doc looks weird. Should `<see cref="IComponent"/>` be replaced with the word `component`?